man/zenfs.8

```diff
@@ -0,0 +1,106 @@
+.\"  SPDX-License-Identifier: GPL-2.0-or-later
+.\"
+.\"  Copyright (C) 2021, Western Digital Corporation or its affiliates.
```

Comment:
Use spdx-tag for copyright as well, see https://github.com/westerndigitalcorporation/libzbd/blob/master/tools/cli/zbd.8